### ISSUE TITLE
Unigram accessability bugs

### ISSUE THREAD

**Issue by OlegTheSnowman** [UnigramDev/Unigram#2999]
Issue 1:
When attempting to bring up the topic list by clicking "Enter" on a group, the screen reader focus is directed to the "Nothing to see" button instead of the topic list. This should be fixed by directing the screen reader focus to the topic list instead, as is the case when opening a chat's edit field.
Issue 2:
When there are unread messages in a topic and the topic is opened, the user is unable to scroll up past the "Unread Messages" line. This should be fixed to allow the user to scroll up to the beginning of the topic if necessary.
Issue 3:
Users using a screen reader are unable to switch to a different account via menu as the buttons are not accessible. This issue should be addressed to ensure that the buttons are accessible and the screen reader does not skip them.
Contacts List and Settings List:
Issue 1:
When opening the list of contacts, the screen reader focus moves to the "Open Navigation Menu" button. This should be fixed by directing the screen reader focus to the contact list to enable users to work more efficiently.
Issue 2:
When opening settings via the menu, the screen reader focus is directed to the "Nothing" button instead of the settings list. This should be fixed by directing the screen reader focus to the settings list for user convenience.
Issue 3:
When opening a setting category by hitting "Enter" on it, nothing happens, and the category just opens. It would be more convenient if the screen reader focus moved to the first option of the opened category, and pressing "Escape" would close the category and return the screen reader focus to the list of the settings categories. This should be fixed to improve user experience.
Issue 4:
Screenreader focuses on open the navigation menu button after opening Unigram.
It would be more convenient if it'd focus on a chat list upon opening for enable users to work faster.
Misk:
Unigram lags the screenreader when downloading something.
When downloading something unigram lags the screenreader kinda much when the chat is opened.
In some cases, i can't even close the chat itself for avoid it and i have to wait til the download finishes.
Hope there's ways to fix it, because sometimes it gets too laggy.
Unigram version: 
Unigram 9.5 (8030) X64 Direct.
Windows version:
Windows 11 22H2 (AMD64) build 22621.1265.

**Comment by MarcAnt01** [comment]
@OlegTheSnowman Please for the future, create one separated ticket for every issue. Having such issues in our repo makes the triage very slow and eventually slow down their fixes

**Comment by MarcAnt01** [comment]
@OlegTheSnowman since we have moved to https://bugs.telegram.org/ we ask you to report all the accessibility bugs there. Please create one card per ticket to ease the triage and don't report accessibility issues regarding to topics, since they are already being tracked in https://github.com/UnigramDev/Unigram/issues/2953
